README.md

```diff
@@ -69,26 +69,21 @@ On ArchLinux:
 sudo pacman -S  base-devel rsync opam pandoc jre8-openjdk mpfr maven
 ```
 
```

Comment:
I'm not sure you need to install XCode. You need just need the command line tools package, which most users would already have.